tests/parsing/test_full_telemetry_parsing.py

```diff
@@ -1,8 +1,75 @@
 import pytest
+from modules.telemetry.v1.block import BlockHeader
+from modules.telemetry.telemetry_utils import parse_radio_block
 from modules.telemetry.telemetry_utils import is_valid_packet_header
 from modules.telemetry.v1.block import PacketHeader
 from modules.misc.config import load_config
 
+LINE1 = "564133494e490000000c0101010000000200020000000000ac5d00000200030000000000f0c300000200030000000000f0c30000"
```

Comment:
This is not a descriptive name for a constant. Should be something like `SAMPLE_PACKET`. It does not matter that this is line 1 of a mission file.